out/namespaces/tabs.d.ts

```diff
@@ -1371,7 +1371,7 @@ export namespace Tabs {
          * first tab in the array instead.
          * @param options Optional.
          */
-        moveInSuccession(tabIds: number[], tabId?: number, options?: MoveInSuccessionOptionsType): void;
+        moveInSuccession(tabIds: number[], tabId?: number, options?: MoveInSuccessionOptionsType): Promise<void>;
```

Comment:
question: This is not listed in the docs as a promise return value (the docs are unclear about this). I'm assuming you tried this yourself or did I miss something?

Reply:
I have reverted this change.

Reply:
I didn't mean to say that it's incorrect. I was just wondering where you got the idea from.